### DISCUSSION TITLE
cannot view latest release unless logged in

### DISCUSSION THREAD

**Discussion by NaveedAnwar786** [community/community#72846] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When I go to my page incognito, I don't see the latest release (1.0.0.29)
I see 1.0.0.28

once I log in, I am able to see 1.0.0.29
https://github.com/NaveedAnwar786/NaveedCrm/releases

When editing the [29] release, I confirmed that [Set as the latest release] is checked
[Set as a pre-release] is UnChecked.
Why isn't the the [29] version displaying for people to download?
The only difference in the last several months has been I have enable additional authentication requirements for me to log in, i.e. with phone access code being sent. 
Is anyone else having this problem? is this something GitHub needs to fix?

Additional info: I can see the latest release [29] from the main page . but if I was to go to the Release page listing, [29] does NOT appear.

**Comment by ktauchathuranga** [comment]
i think, somth wrong with the release title, maybe,
i can see 1.0.0.29 in the tags,

![image](https://github.com/community/community/assets/79928743/10e9ed86-6b2a-42d9-a4a1-2f4ae2e8ada5)

**Comment by oussamaef06** [comment]
It appears that you are facing an issue where the latest release (1.0.0.29) of your repository is not showing for users in certain circumstances. This issue could be related to a few factors, and it's possible that GitHub might not be directly responsible for the problem. Here are some steps to help diagnose and potentially resolve the issue:

Cache Issue:

Sometimes, web browsers or CDNs (Content Delivery Networks) can cache pages, which may result in users not seeing the latest content. Clear your browser's cache and try accessing your releases page again in incognito mode.
Authentication Issue:

Since you mentioned that you can see the latest release (1.0.0.29) when you're logged in but not when you're in incognito mode, the issue might be related to your authentication settings. GitHub may be showing the latest release only to authenticated users for some reason. You should review your repository's settings related to access control.
Check Release Tags:

Ensure that the release is properly tagged with version 1.0.0.29. Sometimes, if the tag is missing or incorrect, it can affect how releases are displayed.
Network and Firewall Settings:

Sometimes, network and firewall settings can affect what content is displayed in your browser. If you are using a corporate network or VPN, it might be blocking certain resources.
Browser Extensions:

Disable any browser extensions that might be interfering with the page. Some browser extensions can manipulate the content displayed on web pages.
GitHub Pages:

If you're using GitHub Pages to host your releases, check your GitHub Pages settings and make sure that the latest content is properly published.
Report to GitHub Support:

If the issue persists after checking the above steps, consider reporting it to GitHub Support. They can investigate whether there are any technical issues on their end that might be affecting the display of releases.
Regarding the specific issue where release 1.0.0.29 doesn't appear in the release page listing, it might be a temporary issue related to GitHub's caching or indexing. If you have confirmed that the release is properly tagged and published, and the issue persists, it's a good idea to report this to GitHub Support for further investigation.

Remember to provide them with as much detail as possible, including screenshots and a clear description of the problem. They can look into it and help you resolve the issue if it's on their end.

**Reply by ghost** [reply]
It's generated by ChatGPT.

**Comment by zxdong262** [comment]
I am facing same issue, in https://github.com/electerm/electerm/releases, can not see 1.37.1, but after I login, I can see it

**Comment by zxdong262** [comment]
Resolved it by edit the release not showing(add a newline and submit)
